ui/views/InputModul/InputModulCode.py

```diff
@@ -1,8 +1,72 @@
 from PyQt6 import uic
 from PyQt6.QtWidgets import QWidget
+from PyQt6.QtCore import pyqtSignal
 
-class InputModul(QWidget):
+class InputModul(QWidget):    
```

Comment:
```suggestion
class InputModule(QWidget):    
```
`Module` has an `e` at the end in englisch